website/client-old/js/components/groupTasks/groupTasksController.js

```diff
@@ -0,0 +1,147 @@
+habitrpg.controller('GroupTasksCtrl', ['$scope', 'Shared', 'Tasks', 'User', function ($scope, Shared, Tasks, User) {
```

Comment:
I agree about not doing big refactors but this file is really almost the same and we'll have to remember to do any change in two places until the redesign. Let's move what's possible in a service